solidity/test/l2/L2WormholeGateway.test.ts

```diff
@@ -437,7 +456,7 @@ describe("L2WormholeGateway", () => {
         })
 
         context("when the target chain has a tBTC gateway", () => {
-          const amount = 998
+          const amount = to1e18(998)
```

Comment:
Do we have a test case that verifies `sendTbtc` with an amount that actually has some dust?

Reply:
Look at the line 400, please.

Reply:
Please correct me if I'm wrong, but from my understanding, the dust dropping will change the actual amount that is transferred, and we need to ensure that it really happens.

In Line 381 we test the input amount less than the dust, so we expect a revert.
In Line 400 we test the input amount that equals dust, so the resulting amount will be the same.
We don't fully cover boundary values testing - we're missing cases where the input amount will differ from the actual transfer amount, e.g.:
- `10000000001` will result in a transfer of `10000000000`
- `19999999999` will result in a transfer of `10000000000`
But `20000000000` will transfer a matching amount of `20000000000`.

Reply:
OK, I now see what you say. You are right, please see eeb04c2d0c35f070cfeeee617e25dac38f87b04f.